src/wix/WixToolset.Core.ExtensionCache/ExtensionCacheManager.cs

```diff
@@ -29,12 +30,17 @@ public ExtensionCacheManager(IMessaging messaging, IExtensionManager extensionMa
         {
             this.Messaging = messaging;
             this.ExtensionManager = extensionManager;
+
+            var fileVersion = FileVersionInfo.GetVersionInfo(typeof(ExtensionCacheManager).Assembly.Location);
+            this.WixVersion = fileVersion.ProductMajorPart.ToString();
```

Comment:
Another option is to use `typeof(ExtensionCacheManager).Assembly.GetType().Name.Version.Major` (or something a lot like that). That gives the AssemblyVersion instead of FileVersion.